lib/liquid/extensions.rb

```diff
@@ -25,6 +26,12 @@ def to_liquid
   end
 end
 
+class BigDecimal
+  def to_liquid
+    to_f
```

Comment:
yea exactly, the `to_s` thing is the reason why I didn't want to just do `Utils.to_number(input).abs`